refinery/ui/cypress/support/commands.js

```diff
@@ -36,7 +36,7 @@ function shell_quote(str) {
 
 Cypress.Commands.add('wrap_cmd',
   function(manage_cmd) {
-    var cd_cmd = "cd .. && " + manage_cmd;
+    var cd_cmd = "cd .. && source ~/virtualenv/python3.5.6/bin/activate && " + manage_cmd;
```

Comment:
Appears to be redundant with `workon refinery-platform`

Reply:
this is specifically for travis - the `workon` command is not run because vagrant is not run

Reply:
It was working previously, so why wouldn't `workon` run now? What is the connection between virtualenvwrapper and Vagrant exactly?

Reply:
The `workon_cmd` is not run in travis, but the `cd_cmd` is.  I have not tried it for a bit, and so I could be wrong, but it appeared that the virtualenv was not persisting to the cypress test for some reason.  Thus it needed to be reactivated.

Reply:
It is still not clear why did this addition become necessary. What was the error message? Also, that path does not exist in Vagrant:
```shell
vagrant@refinery:~$ ls ~/virtualenv/python3.5.6/bin/activate
ls: cannot access '/home/vagrant/virtualenv/python3.5.6/bin/activate': No such file or directory
```

Reply:
The build fails in `travis` without it, if my memory serves.  I'll checkout a separate branch to confirm and remove the line to see  what happens.  If it is necessary, I'll move it so that it executes separate from the other commands.